resources/models/resource.py

```diff
@@ -205,7 +214,9 @@ class Resource(ModifiableModel, AutoIdentifiedModel):
     reservation_info = models.TextField(verbose_name=_('Reservation info'), null=True, blank=True)
     responsible_contact_info = models.TextField(verbose_name=_('Responsible contact info'), blank=True)
     generic_terms = models.ForeignKey(TermsOfUse, verbose_name=_('Generic terms'), null=True, blank=True,
-                                      on_delete=models.SET_NULL)
+                                      on_delete=models.SET_NULL, related_name='resources_where_generic_terms')
+    payment_terms = models.ForeignKey(TermsOfUse, verbose_name=_('Payment terms'), null=True, blank=True,
+                                      on_delete=models.SET_NULL, related_name='resources_where_payment_terms')
     specific_terms = models.TextField(verbose_name=_('Specific terms'), blank=True)
```

Comment:
Should `specific_terms` be refactored as a type of `TermsOfUse` class as well?

Reply:
`generic_terms` and `payment_terms` uses predefined set of terms defined in Django Admin. These predefined terms can then be chosen to certain resource from a drop down menu in Respa Admin by admins and managers. Same terms are used for multiple resources.

`specific_terms` does not pick its value from any predefined set of terms because they are specific for a certain resource. Therefore defining them separately as TermsOfUse objects would in my opinion only add complexity and no positive effects that I can think of.

Reply:
Thanks for clarification. Seems reasonable. Approved :)